server/feeders/sensorsupdater.js

```diff
@@ -54,19 +54,18 @@ var feedIDASSensors = function() {
 
   // Limit the number of calls to be done in parallel
   var q = async.queue(function(task, callback) {
```

Comment:
are we using async library together with promises? can we drop async and stick to promises? that would make this code less messy